templates/federated_reporting/import_file.sh

```diff
@@ -28,15 +28,12 @@ mv "$file" "$file.importing"
   cat<<EOF
 \set ON_ERROR_STOP 1
 BEGIN;
-SELECT ensure_feeder_schema('$hostkey', ARRAY[$table_whitelist]);
+SELECT switch_to_feeder_schema('$hostkey'); -- so that the import below imports into it
 EOF
 
   "$CFE_FR_EXTRACTOR" $CFE_FR_EXTRACTOR_ARGS "$file.importing" | sed $CFE_FR_SED_ARGS $(printf ' -f %s' $sed_scripts)
 
-cat<<EOF
-
-SET SCHEMA 'public';
-SELECT attach_feeder_schema('$hostkey', ARRAY[$table_whitelist]);
+  cat<<EOF
```

Comment:
Let's put the "handle import failure and rollback hub_n_old schema name change" code in the ` || { ... } ` block below around line 42.

Reply:
No, as explained above.